nextflow.config

```diff
@@ -76,6 +76,9 @@ params {
     shortread_complexityfilter_prinseqplusplus_dustscore = 0.5
     save_complexityfiltered_reads                        = false
 
+    // run merging
+    run_merging                = false
+    save_runmerged_reads       = false
```

Comment:
I'm not sure about this parameter name. In my mind `run_` was reserved for the actual profilers. I agree, though, that the name does make sense for what it should do.

Reply:
Ok, what about `perform_*`?

I could then change all the other pre-rpcessing workflows to the same scheme?

perform_complexityfiltering, perform_hostremoval etc.?

Reply:
I would rather have a consistent 'action' verb for all subworkflows, as I find I'm more likely to remember it does something more 'major'? What don't you like about 

```
--perform_complexityfiltering --perform_hostremoval --perform_runmerging
```

Reply:
I do like those 👍🏼

Reply:
Done!